Sources/Basic/Process.swift

```diff
@@ -267,6 +286,30 @@ public final class Process: ObjectIdentifierProtocol {
             throw Process.Error.missingExecutableProgram(program: arguments[0])
         }
 
+    #if os(Windows)
+        _process = Foundation.Process()
+        _process?.arguments = arguments
+        _process?.executableURL = URL(fileURLWithPath: arguments[0])
+
+        if outputRedirection.redirectsOutput {
+            let stdoutPipe = Pipe()
+            let stderrPipe = Pipe()
+            stdoutPipe.fileHandleForReading.readabilityHandler = { (fh : FileHandle) -> Void in
+                let contents = fh.readDataToEndOfFile()
+                self.outputRedirection.outputClosures?.stdoutClosure(contents.map{$0})
```

Comment:
I think you can convert using `[UInt8](contents)`?

Reply:
That makes much for sense to do. Changed!